storage/entities/account.js

```diff
@@ -325,22 +325,34 @@ class Account extends BaseEntity {
 	/**
 	 * Create account object
 	 *
-	 * @param {Object} data
-	 * @param {Object} [options]
-	 * @param {Object} tx - Transaction object
+	 * @param {Object|Array.<Object>} data
+	 * @param {Object} [_options]
+	 * @param {Object} [tx] - Transaction object
 	 * @return {*}
 	 */
 	create(data, _options, tx) {
-		const objectData = _.defaults(data, defaultCreateValues);
-		const createSet = this.getValuesSet(objectData);
-		const attributes = Object.keys(data)
+		let values;
+
+		if (Array.isArray(data)) {
+			values = data;
+		} else if (typeof data === 'object') {
+			values = [data];
+		}
```

Comment:
Even though `values` is an auxiliary variable it will mutate the value in `data`.
To fix that you need something like:
```
if (Array.isArray(data)) {
	values = data.map(item => ({ ...item }));
} else if (typeof data === 'object') {
	values = [{ ...data }];
}
```

Reply:
We are not re-assigning or changing `data`, then how it would be mutated? can't understand.

Reply:
In JS arrays and objects are pointers meaning when you do `values = data` you are assigning the `data` pointer to `value`.